internal/rbd/controllerserver.go

```diff
@@ -833,6 +878,48 @@ func (cs *ControllerServer) DeleteVolume(
 	}
 	defer cs.OperationLocks.ReleaseDeleteLock(volumeID)
 
+	if isMigrationVolHandle(volumeID) {
```

Comment:
move this complete logic to migration.go?

Reply:
Done.